src/main/java/com/microsoft/azure/vmagent/AzureVMAgentCleanUpTask.java

```diff
@@ -244,11 +244,9 @@ public void cleanLeakedResources(
             final Azure azureClient  = TokenCache.getInstance(servicePrincipal).getAzureClient();
             //can't use listByTag because for some reason that method strips all the tags from the outputted resources (https://github.com/Azure/azure-sdk-for-java/issues/1436)
             final PagedList<GenericResource> resources = azureClient.genericResources().listByGroup(resourceGroup);
-
             if (resources == null || resources.isEmpty()) {
```

Comment:
Thanks. I added an addtional check resoures.size() <1 before then remove it. Forget to recover the whitespace, fix it